### ISSUE TITLE
Handle make targets with `/` correctly

### ISSUE THREAD

**Issue by ncfavier** [scop/bash-completion#858]
## Describe the bug

When completing `make foo/<Tab>`, a target named `foo/bar` gets listed as just `bar`, and completing it removes the `foo/` prefix.

Also, completing `foo<Tab>` inserts a slash and a space. It should not insert a space, just like filename completion works.

## To reproduce

1. `git clone --depth=1 https://github.com/NixOS/nix`
2. `cd nix`
3. `make tests/<Tab>`

## Versions (please complete the following information)

- [x] Operating system name/distribution and version: NixOS unstable
- [x] bash version, `echo "$BASH_VERSION"`: 5.1.16(1)-release
- [x] bash-completion version, `(IFS=.; echo "${BASH_COMPLETION_VERSINFO[*]}")`: 2.11

**Comment by akinomyoga** [comment]
Thanks for the report! I believe this is a duplicate of #544. A fix is proposed at #546 but is blocked by an issue related to the test framework. Maybe I should take a look at it again and decide to revert the feature introduced in commit 39f00f92e52b783e7e9e43aac4b4274cc9dee152 that attempts to show the completion list in a compact form.

**Comment by ncfavier** [comment]
Thanks, feel free to close.

**Comment by akinomyoga** [comment]
Thank you for your reply! If you have further comments later, you can post them in #544. Thank you!
